app/data/2_0001.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
It is the id that RRM Classic provides; '2' is agreed because the form type (0001) clashes with UKIS